gr-dtv/lib/dvbt/dvbt_viterbi_decoder_impl.h

```diff
@@ -101,13 +103,13 @@ class dvbt_viterbi_decoder_impl : public dvbt_viterbi_decoder
     int mettab[2][256];
```

Comment:
I know this has fixed size, but  for consistency, wouldn't it be better to use a `std::vector` here as well?

Reply:
Maybe `std::array<>` would be even better.

I'd say so, for another PR. :-)